# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/libdav1d/asm/moz.build
# Commit: 8ce9905a2eec
# Full Hash: 8ce9905a2eec887a9b4ed63798961cef1fa3fded
# Author: Alex Chronopoulos <achronop@gmail.com>
# Date: 2019-04-20 09:35:22
# Regressor Bug: 1540830
# File Overlap Count: 1
# Description:
#   Bug 1540830 - Update build files after new import. r=TD-Linux
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D28201
# ==============================================================================

diff -r a5108ffbd311 -r 8ce9905a2eec media/libdav1d/asm/moz.build
--- a/media/libdav1d/asm/moz.build	Fri Apr 19 20:36:10 2019 +0000
+++ b/media/libdav1d/asm/moz.build	Fri Apr 19 20:36:43 2019 +0000
@@ -83,10 +83,11 @@
             '../../../third_party/dav1d/src/x86/loopfilter.asm',
             '../../../third_party/dav1d/src/x86/looprestoration.asm',
             '../../../third_party/dav1d/src/x86/mc.asm',
+            '../../../third_party/dav1d/src/x86/msac.asm',
         ]
 
     SOURCES += [
-        '../../../third_party/dav1d/src/x86/cdef_ssse3.asm',
+        '../../../third_party/dav1d/src/x86/cdef_sse.asm',
         '../../../third_party/dav1d/src/x86/cpuid.asm',
         '../../../third_party/dav1d/src/x86/ipred_ssse3.asm',
         '../../../third_party/dav1d/src/x86/itx_ssse3.asm',